pom.xml

```diff
@@ -65,7 +65,7 @@
     <version.org.apache.sshd>1.6.0</version.org.apache.sshd>
     <!-- Custom Freemarker build with workaround for random concurrent access issue (annotation processors). See AF-600 for more info.-->
     <version.org.freemarker>2.3.26.jbossorg-1</version.org.freemarker>
-    <version.org.jboss.errai>4.2.2.Final</version.org.jboss.errai>
+    <version.org.jboss.errai>4.2.3-SNAPSHOT</version.org.jboss.errai>
```

Comment:
[16:27:03]  ederign:	tbento can we move 7.7.x errai version to <version.org.jboss.errai>4.2.3-SNAPSHOT</version.org.jboss.errai> ?
[16:27:07]  ederign:	it's safe?
 tbento:	yes, that's the current version on errai's 4.2.x branch. but before we generate a release of the workbench we need to generate 4.2.3.Final